npm-packages/pyoes/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "pyoes",
+  "version": "0.0.2",
```

Comment:
deze versie misschien beter gelijk houden met de versie van pyoes zelf?

Reply:
hmm not sure, op zich staat deze los van de pip package